gpu-operator/release-notes.rst

```diff
@@ -34,6 +34,22 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
 
 ----
 
+.. _v24.6.2:
+
+24.6.2
+======
+
+.. _v24.6.2-new-features:
+
+New Features
+------------
+
+* Added support for the following software component versions:
+
+  - NVIDIA Container Toolkit v1.16.2.
+  - NVIDIA DCGM Exporter v3.3.8-3.6.0
+  - NVIDIA DCGM v3.3.8-1
```

Comment:
The dcgm / dcgm-exporter versions were not changed as part of this release. The versions have been bumped in main, but not in the `release-24.6` branch.

Reply:
TY.  How fortunate for me that it was a separate commit!